std/stdio.d

```diff
@@ -4173,9 +4182,9 @@ en.cppreference.com/w/c/io#Narrow_and_wide_orientation,
 orientation).
 */
 void toFile(T)(T data, string fileName)
-    if (is(typeof(std.algorithm.mutation.copy(data, stdout.lockingBinaryWriter))))
+    if (is(typeof(copy(data, stdout.lockingBinaryWriter))))
 {
-    std.algorithm.mutation.copy(data, File(fileName, "wb").lockingBinaryWriter);
+    copy(data, File(fileName, "wb").lockingBinaryWriter);
```

Comment:
Made these two a module-level import up top, as the fact that this works without an explicit static import is actually a compiler bug that will soon be enforced.